doc/esmvalcore/preprocessor.rst

```diff
@@ -872,6 +872,10 @@ Parameters:
       Available periods: 'full', 'season', 'seasonal', 'monthly', 'month',
       'mon', 'daily', 'day'. Default is 'full'
 
+    * reference: Time slice to use as the reference to compute the climatology
+      on. Can be 'null' to use the full cube or a dictionary with the
+      parameters from extract_time_. Default is null
```

Comment:
It would be nice if you could add an example of how to use this below, because this is not entirely trivial.

Reply:
Done